docs/fileformat-prose.md

```diff
@@ -148,14 +203,17 @@ DataFrames are saved as a columnar format in a group, so each column of a DataFr
 We save a little more information in the attributes here.
 
 ```python
```

Comment:
I'd consider adding the HDF5 + Zarr example also for dataframes, but it's quite straighforward, so this could be addressed even just by saying "each column of a DataFrame is saved as a separate array (e.g. /obs/n_genes_by_counts)"